src/goto-instrument/cover.cpp

```diff
@@ -161,14 +161,20 @@ void coverage_goalst::add_goal(source_locationt goal)
   existing_goals.push_back(goal);
 }
 
-bool coverage_goalst::is_existing_goal(source_locationt source_location) const
+/// compare the value of the current goal to the existing ones
+/// \par source_locationt: source location of the current goal
```

Comment:
Sorry to be picky, but `\par` should better be replaced by `param` and `\par` the parameter name is `source_loc`, not `source_locationt`.